database/migrations/2022_03_15_074131_create_fakturownia_order_table.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateFakturowniaOrderTable extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('fakturownia-orders', function (Blueprint $table) {
```

Comment:
I'm pretty sure we use `_` in table names and not `-`

Reply:
done